api/net/appsettings.Development.json

```diff
@@ -26,5 +26,10 @@
   },
   "SignalR": {
     "EnableDetailedErrors": true
+  },
+  "Reporting": {
```

Comment:
Configuration for reporting within the API.  These configuration values are passed to the template so that it can use them to have the correct environmental URLs